terraform-aws-github-runner/modules/runner-binaries-syncer/lambdas/runner-binaries-syncer/yarn.lock

```diff
@@ -10,6 +10,583 @@
     "@jridgewell/gen-mapping" "^0.3.5"
     "@jridgewell/trace-mapping" "^0.3.24"
 
+"@aws-crypto/crc32@5.2.0":
```

Comment:
nit: I wonder if we can add these to the linguist file at the root to note that these are generated files and should be automatically minimized in PR reviews.

This is unrelated to the migration work but just a thought I was having.